qiskit_ibm_provider/transpiler/passes/scheduling/block_base_padder.py

```diff
@@ -186,7 +188,17 @@ def _visit_delay(self, node: DAGNode) -> None:
         rather than instruction. Delay node is not added so that
         we can extract non-delay predecessors.
         """
-        pass
+        block_idx, t0 = self._node_start_time[node]  # pylint: disable=invalid-name
+        # Trigger the end of a block
+        if block_idx > self._current_block_idx:
+            self._terminate_block(self._block_duration, self._current_block_idx, node)
+
+        self._conditional_block = bool(node.op.condition_bits)
+
+        self._current_block_idx = block_idx
+
+        t1 = t0 + node.op.duration  # pylint: disable=invalid-name
```

Comment:
I noticed I had misunderstanding about conditional delay. This is likely not the scope of this PR, but for example a conditional X gate can be understood as (if my understanding is correct)

- condition = True: Apply X gate of duration 160 dt
- condition = False: Apply delay of duration 160 dt, e.g. turn off port output

and this is an implicit assumption in the current scheduling logic, i.e. conditional operation doesn't change instruction duration. However, in case of conditional delay,

- condition = True: Apply delay of duration 100 dt
- condition = False: Apply what? delay of duration 100 dt? eliminate itself from dag?

If we want to eliminate delay, the backend should dynamically reschedule instructions and update event table -- this is not currently supported I guess. So we should exclude `Delay` from `CONDITIONAL_SUPPORTED`.